extensions/kafka/src/main/java/io/deephaven/kafka/ingest/JsonNodeUtil.java

```diff
@@ -498,11 +498,19 @@ public static Character getBoxedChar(@NotNull final JsonNode node, @NotNull fina
     }
 
     @Nullable
-    public static Character getBoxedChar(final JsonNode tmpNode) {
-        if (isNullOrMissingField(tmpNode)) {
+    public static Character getBoxedChar(final JsonNode node) {
```

Comment:
Why don't our boxed cases delegate to the unboxed and then box the result?

Reply:
Ostensibly to be more "efficient" and save a NULL_CHAR comparison. I can update these code paths to use `TypeUtils.box`?

Reply:
I don't see any callers of the boxed methods, but they are part of the public API...